CMakeLists.txt

```diff
@@ -108,6 +112,7 @@ list(APPEND CMAKE_PREFIX_PATH /opt/rocm /opt/rocm/llvm /opt/rocm/hip /opt/rocm/h
 
 option(ENABLE_HIP_WORKAROUNDS Off)
 set(MIOPEN_INSTALL_CXX_HEADERS Off CACHE BOOL "Install MIOpen's C++ header interface")
+set(MIOPEN_LINK_DB "" CACHE STRING "Colon separated list of of architecture CU pairs to link on-disk DBs in the binary. Example gfx906_60:gfx900_56")
```

Comment:
AFAICS This also switches OFF installing and using the on-disk system-find-db and the system-perf-db. The specified subset of system-find-databases will be available for the library (but the system-perf-db will be fully available).

This should ne also documented.